org.gimp.GIMP.json

```diff
@@ -1,5 +1,5 @@
 {
-    "app-id": "org.gimp.GIMP",
+    "app-id": "org.gimp.GIMP.dev",
     "branch": "beta",
     "runtime": "org.gnome.Platform",
     "runtime-version": "3.36",
```

Comment:
I recommend bump the runtime version

Reply:
it's already at 40

Reply:
Update this development branch, what did i mean, merge master to this branch